rules/no-empty-file.js

```diff
@@ -15,9 +15,17 @@ const isTripleSlashDirective = node =>
 const hasTripeSlashDirectives = comments =>
 	comments.some(currentNode => isTripleSlashDirective(currentNode));
 
+const isProgramFileEmpty = node => node.type === 'Program' && node.body.length === 0;
+
+const isAllowOnlyCommentsFile = (option, node) => option.allow.includes('comments') && isProgramFileEmpty(node);
```

Comment:
I have difficult to understand this, shouldn't it be something like `allowComments && hasComment`?

Reply:
I feel `/* comment */ {}` won't pass, but it should.

Reply:
Thanks for review! I'll check it soon.

Reply:
Done